docs/ManualDeploymentGuideScalarDBServerOnEKS.md

```diff
@@ -0,0 +1,200 @@
+# Deploy Scalar DB Server on AWS
+
+The Scalar DB server is a standalone gRPC server that implements the Scalar DB interface.
+We can deploy the Scalar DB server on any Kubernetes service.
+This guide will help you to deploy a Scalar DB server in AWS EKS.
+
+## What we create
+
+![image](images/scalardbserver-eks-diagram.png)
+
+In this guide, we will create the following components.
+
+* A VPC with NAT gateway
+* An EKS cluster with two Kubernetes node groups
+* A managed database service
+    * DynamoDB
+* A Bastion instance with a public IP
+* Amazon CloudWatch
+
+
+## Step 1. Configure your network
+
+Configure a secure network according to your organizational standard or application requirements. This section shows how to configure a secure network for Scalar DB server deployment.
+
+### Requirements
+
+* You must create VPC with public NAT gateways on private networks. NAT gateway is necessary to enable internet access for Kubernetes node group subnets.
+* You must create at least 2 subnets for the EKS cluster in different availability zones. It is mandatory to create an EKS cluster.
+
+### Recommendations
+
+* You should create private subnets for the Kubernetes cluster for production.
+* You should create a bastion server to manage the Kubernetes cluster.
+* You should create 3 subnets in 3 availability zones for the Kubernetes cluster for higher availability.
+* You should choose public subnets when creating an EKS cluster for public access.
+* You should create subnets with the prefix at least `/24` for the Kubernetes cluster to work without issues even after scaling.
+
+### Steps
+
+* Create an Amazon VPC based on the [AWS official guide](https://docs.aws.amazon.com/vpc/latest/userguide/working-with-vpcs.html) with the above requirements and recommendations.
```

Comment:
```suggestion
* Create an Amazon VPC based on the [AWS official guide](https://docs.aws.amazon.com/eks/latest/userguide/creating-a-vpc.html) with the above requirements and recommendations.
```
The EKS official document explains how to create VPC for EKS.
So, I think it is better to refer to this document.

Reply:
Thank you.
I have updated the link.